### PR TITLE
fix: ethrpc: Add missing fields to EthTx

### PR DESCRIPTION
**Pull request by magik6k**
## Related Issues
Closes https://github.com/filecoin-project/lotus/issues/10106

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [x] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: chain/types/ethtypes/eth_transactions.go [hunk 1]
```diff
@@ -37,7 +37,8 @@ type EthTx struct {
 	Gas                  EthUint64   `json:"gas"`
 	MaxFeePerGas         EthBigInt   `json:"maxFeePerGas"`
 	MaxPriorityFeePerGas EthBigInt   `json:"maxPriorityFeePerGas"`
-	V                    EthBigInt   `json:"v"`
+	AccessList           []EthHash   `json:"accessList"`
+	V                    EthBigInt   `json:"yParity"`
```
Comment (magik6k) [inline chain/types/ethtypes/eth_transactions.go:41]: Afaict what we put here is in fact YParity according to https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2098.md; V is only used for legacy transactions

This value seems to be set to 0/1; not 27/28 which are V values

Not sure if it makes sense to rename the V field to `YParity` as well, no strong opinions.

File: chain/types/ethtypes/eth_transactions.go [hunk 2]
```diff
@@ -37,7 +37,8 @@ type EthTx struct {
 	Gas                  EthUint64   `json:"gas"`
 	MaxFeePerGas         EthBigInt   `json:"maxFeePerGas"`
 	MaxPriorityFeePerGas EthBigInt   `json:"maxPriorityFeePerGas"`
-	V                    EthBigInt   `json:"v"`
+	AccessList           []EthHash   `json:"accessList"`
```
Comment (magik6k) [inline chain/types/ethtypes/eth_transactions.go:40]: Those come from https://eips.ethereum.org/EIPS/eip-2930, which I don't think are supported in FEVM

Given that those aren't possible to specify (?), and optional, I just add the field and keep it empty, which is enough to keep openrpc happy.

**Review by arajasek — APPROVED** [review]
Changes themselves look good to me, but review from @ychiaoli18 / @raulk / Aadi would be good

File: chain/types/ethtypes/eth_transactions.go (hunk 1 above)
Comment (ychiaoli18) [reply chain/types/ethtypes/eth_transactions.go:41]: I'd prefer to rename it to yParity as well to keep them consistent.

**Review by ychiaoli18 — APPROVED** [review]

**Pull request merged**